app/helpers/application_helper.rb

```diff
@@ -61,8 +61,8 @@ def t_fallback(key, options = {})
     end
   end
 
-  def render_govspeak(content)
-    render "govuk_publishing_components/components/govspeak" do
+  def render_govspeak(content, inverse = nil)
```

Comment:
```suggestion
  def render_govspeak(content, inverse: false)
```

Since this is a boolean it's probably better to set this as false, setting it as a keyword argument will make the code that calls it a bit easier to understand too as to what the true means.

Reply:
Maybe you know the answer to this - I tried this syntax originally (`inverse:` instead of `inverse =`) and I got a massive page error of `wrong number of arguments (given 2, expected 1)`, so I went with `=` instead. Any idea what the difference is?

Reply:
I've changed it to `false` though as that made sense.

Reply:
Oh. Wait. Changing it to false causes a linting error. Weird that `inverse = false` causes a lint error but `inverse = nil` doesn't. Or maybe I'm going mad.

Reply:
Sure, the linter doesn't realise it's meant to be a boolean when you set it to nil so it doesn't complain.

I think the issue you've got is that once you change to `def render_govspeak(content, inverse: false)`

Then you'll also need to change `<%= render_govspeak(header["intro"], true) %>` to `<%= render_govspeak(header["intro"], inverse: true) %>` - sorry I should have pointed that out. This makes this render_govspeak call easier to understand since the boolean has an associated key.

Reply:
Ah, of course. Thanks, have changed.